torch/csrc/distributed/rpc/rref_context.cpp

```diff
@@ -293,21 +301,26 @@ void RRefContext::notifyOwnerAndParentOfFork(
     return;
   }
 
+  rpc::RpcRetryOptions options;
+  options.maxRetries = 20;
```

Comment:
As this appears in multiple places, I wonder if we should wrap it and the subsequent `sendWithRetries` into a utility function in `rref_context.h/cpp`